installers-exp/spyder-app/meta.yaml

```
@@ -0,0 +1,119 @@
+{% set name = "spyder" %}
+{% set version = "5.4.0.dev0" %}
+{% set build = 0 %}
+
+package:
+  name: spyder
```

```suggestion
  name: {{ name }}
```

Wouldn't it make sense to use `name` here (and elsewhere), since you've defined it above?

---

I've taken a lot from napri's workflow, with my primary goal to get something that "just works". My latest push should reflect a better rendition of how I want all the code organized, so please feel free to take another look at this and any other issues you find. Thanks for your help and feedback.